.env

```diff
@@ -22,3 +22,4 @@ SITE_NAME=''
 SUPPORT_EMAIL=''
 SUPPORT_URL=''
 USER_INFO_COOKIE_NAME=''
+ENABLE_PROGRESS_GRAPH_SETTINGS=false
```

Comment:
here the value is bool I think and when it will be true will it contain a true as string or just true as bool value?

Reply:
yes it looks like a boolean value but eventually, it will be used as a string because every value in env is available to js as a srting.